stores/yaml/store.go

```diff
@@ -101,6 +101,8 @@ func (store Store) treeBranchToYamlMap(in sops.TreeBranch) yaml.MapSlice {
 	return branch
 }
 
+// LoadEncryptedFile loads the contexts of an encrypted yaml file onto a
```

Comment:
Did you mean to write "contents" rather than "context"? Also present in the comment below.